deploy/README.md

```diff
@@ -0,0 +1,8 @@
+# Deploying the search client to a custom Wordpress page template
+
+1. Run `npm run build` to build this project.
+2. Figure out which Wordpress theme you're using and navigate to the directory for that theme under wp\_content.
+3. Add the code in ourroots\_cms\_functions.php to the end of the functions.php file in that directory. If you are upgrading a previous installation, you may want to change the version numbers from 0.0.1 to something else for cache-busting.
```

Comment:
'that directory' refers to the one in Wordpress under `wp_content`?

Reply:
yes, I clarified this.